include/diagnostics/psrf.hpp

```diff
@@ -0,0 +1,62 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2012-2020 Vissarion Fisikopoulos
+// Copyright (c) 2018-2020 Apostolos Chalkis
+
+//Contributed and/or modified by Alexandros Manochis, as part of Google Summer of Code 2020 program.
+
+//Licensed under GNU LGPL.3, see LICENCE file
+
+
+#ifndef PSRF_HPP
+#define PSRF_HPP
+
+template <typename NT, typename VT, typename MT>
+NT perform_psrf(MT const& points)
```

Comment:
here there are not frac1, frac2 and the return type is NT compared to bool above

Reply:
I have added comments to provide information about psrf diagnostic